Tests/PackageModelSyntaxTests/ManifestEditTests.swift

```diff
@@ -564,6 +564,53 @@ class ManifestEditTests: XCTestCase {
         }
     }
 
+    func testAddSwiftTestingTestTarget() throws {
+        try assertManifestRefactor("""
+            // swift-tools-version: 5.5
+            let package = Package(
+                name: "packages"
+            )
+            """,
+            expectedManifest: """
+            // swift-tools-version: 5.5
```

Comment:
Where does 5.5 actually come from 🤔?

Reply:
5.5 introduced some tweaks to the package manifest format (e.g., some things moved to labels), such as introducing new `package` dependencies and deprecating some old ones. We can either write a bunch of code to deal with all of the differences, or we can limit the functionality to 5.5 or newer. The latter was easier.

Reply:
swift-testing is going to need a manifest version of 6.0 or later at some point in the future.

Reply:
(I'm not sure if that affects its _clients_—I assume not?)

Reply:
Yeah, if we end up changing the manifest format in some way for swift-testing, we'll need make changes here as well. We can enforce a 6.0 manifest or help with updating the manifest.